pup/utils/configuration.py

```diff
@@ -11,3 +11,4 @@
 AWS_ACCESS_KEY_ID = os.getenv('CW_AWS_ACCESS_KEY_ID', None)
 AWS_SECRET_ACCESS_KEY = os.getenv('CW_AWS_SECRET_ACCESS_KEY', None)
 AWS_REGION_NAME = os.getenv('CW_AWS_REGION_NAME', 'us-east-1')
+SYSTEM_PROFILE_QUEUE = os.getenv("SYSTEM_PROFILE_QUEUE", "platform.system-profile")
```

Comment:
shouldn't this be `platform.upload.system-profile` or `platform.pup.system-profile`? Unless we don't have a clear naming convention. I thought everything was to be 3 sections.